### PR TITLE
feat: expose SERVICE in instrument as INSTRUMENT_CHANNEL

### PR DESCRIPTION
**Pull request by KazuCocoa**
for https://github.com/appium/appium-xcuitest-driver/pull/1403

"SERVICE" could be confusing, so let's define as CHANNEL.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { default as Usbmux } from './lib/usbmux';
 import * as utilities from './lib/utilities';
 import * as services from './lib/services';
+import { SERVICE as INSTRUMENT_SERVICE } from './lib/instrument';
```
Comment (mykola-mokhnach) [inline index.js:4]: I'm not quite sure where this is exported. Shouldn't all services be started using start...Service helpers?

Comment (KazuCocoa) [reply index.js:4]: This usage is over `services.startInstrumentService`.

e.g. https://github.com/appium/appium-xcuitest-driver/pull/1403/files#diff-cadf38942bfa2205c3e85f7e37de81f7dc552069916c98d45b2be3b06ab4d544R97

```
    const instrumentService = await services.startInstrumentService(this.opts.device.udid);
    const processes = await instrumentService.callChannel(SERVICE.DEVICE_INFO, 'runningProcesses');
```

So, this is not service itself, but used over service.
Hm... I wonder `CHANNEL`, `INSTRUMENT_CHANNEL` is better naming than `SERVICE` to reduce confusion.

currently the xcuitest driver refer to this value via `import { SERVICE } from 'appium-ios-device/build/lib/instrument';`, but probably `import { services, INSTRUMENT_SERVICE } from 'appium-ios-device';` in xcuitest more make sense...

Comment (KazuCocoa) [reply index.js]: renamed.

File: lib/instrument/index.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ const CHANNEL_OFFSET = 2 ** 32;
 const INSTRUMENT_SERVICE_NAME_VERSION_14 = 'com.apple.instruments.remoteserver.DVTSecureSocketProxy';
 const INSTRUMENT_SERVICE_NAME = 'com.apple.instruments.remoteserver';
 
-export const SERVICE = Object.freeze({
+export const CHANNEL = Object.freeze({
```
Comment (mykola-mokhnach) [inline lib/instrument/index.js:20]: would this be a breaking change? Why cannot we just add INSTRUMENT_CHANNEL to the list of other channels?

Comment (KazuCocoa) [reply lib/instrument/index.js:20]: No, only test code in this package refers to this for now.

Just thought `instrument` was redundant as the original definition's intention. Let me update.

Comment (KazuCocoa) [reply lib/instrument/index.js]: ah, if someone referred to this value over `appium-ios-device/build/lib/instrument`, could be a breaking change. Our modules do not have any references though.

**Review by mykola-mokhnach — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (mykola-mokhnach) [reply index.js]: I like CHANNEL better over INSTRUMENT_CHANNEL, but both would work

**Pull request merged**